### PR TITLE
Use ttag to provide i18n for critical path

### PR DESCRIPTION
**Pull request by MrOrz**
This PR adds English translation into LINE bot.

-  Upgrades to Babel 7 for `ttag-cli` to work
- `LOCALE` env now controls the line bot's language.
- Also updates some of the copy in #140 and #141 .

English line bot is connected to LINE ID @439txwho ( http://nav.cx/sM4NFfu )

Screenshot_20191111-141957

Screenshot_20191111-142003

Screenshot_20191111-142008

### CODE REVIEW

File: src/handlers/utils.js [hunk 1]
```diff
@@ -18,10 +19,22 @@ export function createPostbackAction(label, input, issuedAt) {
...
 export function createFeedbackWords(positive, negative) {
-  if (positive + negative === 0) return '[還沒有人針對此回應評價]';
+  if (positive + negative === 0) return `[${t`No feedback yet`}]`;
   let result = '';
-  if (positive) result += `有 ${positive} 人覺得此回應有幫助\n`;
-  if (negative) result += `有 ${negative} 人覺得此回應沒幫助\n`;
+  if (positive)
+    result +=
+      ngettext(
+        msgid`${positive} user consider this helpful`,
```
Comment (changhc) [inline src/handlers/utils.js]:
```suggestion
        msgid`${positive} user considers this helpful`,
```
and a few lines below as well

File: src/handlers/initState.js [hunk 2]
```diff
@@ -107,17 +108,25 @@ export default async function initState(params) {
...
+            ],
           }))
           .concat(
             hasIdenticalDocs
               ? []
               : [
                   {
-                    text: '這裡沒有一篇是我傳的訊息。',
-                    actions: [createPostbackAction('選擇', 0, issuedAt)],
+                    text: t`These messages don't match mine :(`,
```
Comment (changhc) [inline src/handlers/initState.js]:
```suggestion
                    text: t`None of these messages matches mine :(`,
```
sounds better to me

File: src/handlers/choosingArticle.js [hunk 3]
```diff
@@ -153,17 +154,39 @@ export default async function choosingArticle(params) {
...
-      '這個訊息有：\n' +
-      `${count.RUMOR || 0} 則回應標成 ❌ 含有不實訊息\n` +
-      `${count.NOT_RUMOR || 0} 則回應標成 ⭕ 含有真實訊息\n` +
-      `${count.OPINIONATED || 0} 則回應標成 💬 含有個人意見\n` +
-      `${count.NOT_ARTICLE || 0} 則回應標成 ⚠️️ 不在查證範圍\n`;
+      t`Volunteer editors has publised several replies to this message.` +
+      '\n\n👨‍👩‍👧‍👦 ' +
+      [
+        count.RUMOR > 0
+          ? t`${count.RUMOR} of them say it ❌ contains misinformation`
```
Comment (changhc) [inline src/handlers/choosingArticle.js]: Though pretty annoying, should we apply `ngettext` for verbs here as well?

File: src/handlers/initState.js [hunk 4]
```diff
@@ -107,17 +108,25 @@ export default async function initState(params) {
         type: 'carousel',
         columns: edgesSortedWithSimilarity
           .map(({ node: { text }, similarity }, idx) => ({
-            text: `[相似度:${(similarity * 100).toFixed(2) +
-              '%'}] \n ${ellipsis(text, 100, '')}`,
-            actions: [createPostbackAction('選擇此則', idx + 1, issuedAt)],
+            text: ellipsis(
+              `[${t`Similarity`}:${(similarity * 100).toFixed(2) +
```
Comment (changhc) [inline src/handlers/initState.js]:
```suggestion
              `[${t`Similarity`}: ${(similarity * 100).toFixed(2) +
```

File: src/handlers/utils.js [hunk 5]
```diff
@@ -38,15 +51,15 @@ export function createFlexMessageText(text = '') {
...
+      return t`Contains true information`;
     case 'OPINIONATED':
-      return '含有個人意見';
+      return t`Contains personal perspective`;
     case 'NOT_ARTICLE':
-      return '不在查證範圍';
+      return t`Invalid request`;
   }
-  return '回應的狀態未定義！';
+  return 'Undefined';
```
Comment (changhc) [inline src/handlers/utils.js]: Should we apply i18n to this as well?

File: src/handlers/choosingReply.js [hunk 6]
```diff
@@ -61,26 +63,26 @@ export default async function choosingReply(params) {
...
+          `💁 ${t`These messages are provided by some nice volunteers. Please refer to their references and make judgements on your own.`}\n` +
+          (data.foundReplyIds.length > 1
+            ? `🗣️ ${t`There are different replies for the message. Read them all here before making judgements:`}\n${articleUrl}\n`
+            : '') +
+          `\n⁉️ ${t`If you have different thoughts, you may have your say here:`}\n${articleUrl}`,
       },
       {
         type: 'template',
         altText:
           '請問上面回應是否有幫助？\n「是」請輸入「y」，「否」請至手機上回應',
```
Comment (changhc) [inline src/handlers/choosingReply.js]: i18n for this?

File: src/handlers/choosingArticle.js (hunk 3 above)
Comment (MrOrz) [reply src/handlers/choosingArticle.js]: Nope XD

File: src/handlers/choosingReply.js (hunk 6 above)
Comment (MrOrz) [reply src/handlers/choosingReply.js]: Yeah, maybe future commits. Thanks!

File: src/handlers/utils.js (hunk 5 above)
Comment (MrOrz) [reply src/handlers/utils.js]: I think it's ok, we don't need to add a translation entry for a content that only appears when we omit something.

When this appears, it means we are adding more cases and provide proper translation to that case.

**Pull request merged**